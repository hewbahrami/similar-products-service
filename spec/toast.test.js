import React from 'react';
import { render, cleanup } from '@testing-library/react';
// import "@testing-library/jest-dom";
import Toast from '../client/toast';

test('renders', () => {
  const { asFragment } = render(<Toast name="EKTORP" />);
  expect(asFragment()).toMatchSnapshot();
})