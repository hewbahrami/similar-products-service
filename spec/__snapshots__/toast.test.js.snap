// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm lbdhuO"
  >
    <div
      class="sc-AxirZ bKmrVn"
    >
      <div
        class="sc-AxiKw eJlaIa"
      >
        EKTORP was saved to the Shopping Cart.
      </div>
      <button
        class="sc-AxhCb fVhKMx"
      >
        VIEW
      </button>
    </div>
  </div>
</DocumentFragment>
`;
